ISHPermissionKit/Requests/ISHPermissionRequestMotion.m

```diff
@@ -35,19 +36,20 @@ - (void)requestUserPermissionWithCompletionBlock:(ISHPermissionRequestCompletion
     self.motionActivityQueue = [[NSOperationQueue alloc] init];
     [self.activityManager queryActivityStartingFromDate:[NSDate distantPast] toDate:[NSDate date] toQueue:self.motionActivityQueue withHandler:^(NSArray *activities, NSError *error) {
         ISHPermissionState currentState = ISHPermissionStateUnknown;
-        
+
         if (error && (error.domain == CMErrorDomain) && (error.code == CMErrorMotionActivityNotAuthorized)) {
```

Comment:
We can probably reuse this code path for the external error.

We should check the domain using `isEqualToString:`.